packages/lesswrong/server/givingSeason/resolvers.ts

```diff
@@ -106,7 +106,6 @@ defineFeedResolver<number>({
     context: ResolverContext
   }) => {
     const {tagId} = args;
-    // const relevantPostIds = await context.repos.posts.getViewablePostsIdsWithTag(tagId);
```

Comment:
Why not use this for `GivingSeasonTagFeed`?

Reply:
There's no need when we're only fetching posts since posts already know what tags are applied to them - it's only needed for fetching comments, which don't know the tags (and this is a mongo-style resolver so we can't join here).